.travis.yml

```diff
@@ -11,7 +11,7 @@ env:
     - SYMFONY_DEPRECATIONS_HELPER=weak_vendors
     - APP_ENV=test
 
-matrix:
+jobs:
```

Comment:
Minor future-proofing: `jobs` is used for build stages, but it works the same otherwise. Travis CI docs are outdated: https://github.com/travis-ci/docs-travis-ci-com/issues/1500#issuecomment-359389944